src/repository/exercises-repository.ts

```diff
@@ -1304,1527 +1282,3122 @@ export const db: Schema = {
+      presentSimple: {
+        Eu: 'sigo',
+        Tu: 'segues',
+        'Ela/Ele/Você': 'segue',
+        Nós: 'seguimos',
+        'Eles/Elas/Vocēs': 'seguem'
+      },
+      pastPerfect: {
+        Eu: 'segui',
+        Tu: 'seguiste',
+        'Ela/Ele/Você': 'seguiu',
+        Nós: 'seguimos',
+        'Eles/Elas/Vocēs': 'seguiram'
+      }
+    },
+    {
+      english: 'to serve',
+      infinitive: 'servir',
+      presentSimple: {
+        Eu: 'sirvo',
+        Tu: 'serves',
+        'Ela/Ele/Você': 'serve',
+        Nós: 'servimos',
+        'Eles/Elas/Vocēs': 'servem'
+      },
+      pastPerfect: {
+        Eu: 'servi',
+        Tu: 'serviste',
+        'Ela/Ele/Você': 'serviu',
+        Nós: 'servimos',
+        'Eles/Elas/Vocēs': 'serviram'
+      }
+    },
+    {
+      english: 'to suggest',
+      infinitive: 'sugerir',
+      presentSimple: {
+        Eu: 'sugiro',
+        Tu: 'sugeres',
+        'Ela/Ele/Você': 'sugere',
+        Nós: 'sugerimos',
+        'Eles/Elas/Vocēs': 'sugerem'
+      },
+      pastPerfect: {
+        Eu: 'sugeri',
+        Tu: 'sugeriste',
+        'Ela/Ele/Você': 'sugeriu',
+        Nós: 'sugerimos',
+        'Eles/Elas/Vocēs': 'sugeriram'
+      }
+    },
+    {
+      english: 'to try',
+      infinitive: 'tentar',
+      presentSimple: {
+        Eu: 'tento',
+        Tu: 'tentas',
+        'Ela/Ele/Você': 'tenta',
+        Nós: 'tentamos',
+        'Eles/Elas/Vocēs': 'tentam'
+      },
+      pastPerfect: {
+        Eu: 'tentei',
+        Tu: 'tentaste',
+        'Ela/Ele/Você': 'tentou',
+        Nós: 'tentámos',
+        'Eles/Elas/Vocēs': 'tentaram'
+      }
+    },
+    {
+      english: 'to turn',
+      infinitive: 'virar',
+      presentSimple: {
+        Eu: 'viro',
+        Tu: 'viras',
+        'Ela/Ele/Você': 'vira',
+        Nós: 'viramos',
+        'Eles/Elas/Vocēs': 'viram'
+      },
+      pastPerfect: {
+        Eu: 'virei',
+        Tu: 'viraste',
+        'Ela/Ele/Você': 'virou',
+        Nós: 'virámos',
+        'Eles/Elas/Vocēs': 'viraram'
+      }
+    },
+    {
+      english: 'to sell',
+      infinitive: 'vender',
+      presentSimple: {
+        Eu: 'vendo',
+        Tu: 'vendes',
+        'Ela/Ele/Você': 'vende',
+        Nós: 'vendemos',
+        'Eles/Elas/Vocēs': 'vendem'
+      },
+      pastPerfect: {
+        Eu: 'vendi',
+        Tu: 'vendeste',
+        'Ela/Ele/Você': 'vendeu',
+        Nós: 'vendemos',
+        'Eles/Elas/Vocēs': 'venderam'
+      }
+    },
+    {
+      english: 'to bring',
+      infinitive: 'trazer',
+      presentSimple: {
+        Eu: 'trago',
+        Tu: 'trazes',
+        'Ela/Ele/Você': 'traz',
+        Nós: 'trazemos',
+        'Eles/Elas/Vocēs': 'trazem'
+      },
+      pastPerfect: {
+        Eu: 'trouxe',
+        Tu: 'trouxeste',
+        'Ela/Ele/Você': 'trouxe',
+        Nós: 'trouxemos',
+        'Eles/Elas/Vocēs': 'trouxeram'
+      }
+    },
+    {
+      english: 'to become',
+      infinitive: 'transformar-se',
```

Comment:
transformar-se is not a verb
is transformar 
otherwise 'concentrar' also makes more sense to write 'concentrar-me'